manifests/init.pp

```diff
@@ -11,6 +11,7 @@
   $package_ensure,
   $manage_init_defaults = false,
   $manage_package       = true,
+  $manage_repo          = false,
```

Comment:
I think setting this will render the data/ defaults useless.
So maybe remove the false here and put it into data/default.yaml

Reply:
It seems to be working in production, but it hides the fact that some systems will default to `true`… 

I will work on this today.

FYI, there is discussion related to this ATM at voxpupuli (PR43 of voxpupuli/puppet-gitlab_ci_runner… not putting an URL to avoid this issue being linked there: it would not bring new information in the context of this distant PR)

Reply:
I have amended the commit to have the default value in Hiera.